### DISCUSSION TITLE
请问PaddleOCR做车牌识别效果怎么样？有没有尝试过，百度的车牌识别框架是用的哪种呢？

### DISCUSSION THREAD

**Discussion by ctgushiwei** [PaddlePaddle/PaddleOCR#12680] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

**Comment by a31413510** [comment, marked as answer]
您好，PaddleOCR有车牌识别的相关demo，如果自己有数据做简单finetune效果挺不错的，可以参考这个文档：
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/applications/%E8%BD%BB%E9%87%8F%E7%BA%A7%E8%BD%A6%E7%89%8C%E8%AF%86%E5%88%AB.md

**Comment by hellohahaw** [comment]
> 您好，PaddleOCR有车牌识别的相关demo，如果自己有数据做简单finetune效果挺不错的，可以参考这个文档： https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/applications/%E8%BD%BB%E9%87%8F%E7%BA%A7%E8%BD%A6%E7%89%8C%E8%AF%86%E5%88%AB.md
请问双行车牌可以正确识别码？
